src/database/seeders/20210120052931-userRoles.js

```diff
@@ -2,8 +2,12 @@
 module.exports = {
   up: async (queryInterface, Sequelize) => {
     await queryInterface.bulkInsert('userRoles', [{
-      name: process.env.ADMIN_ROLE_NAME,
-      description: process.env.ADMIN_ROLE_DESCRIPTION,
+      // name: process.env.ADMIN_ROLE_NAME,
+      // description: process.env.ADMIN_ROLE_DESCRIPTION,
+      // createdAt: new Date(),
+      // updatedAt: new Date()
+      name: 'MEMBER',
+      description: 'This is a member',
```

Comment:
Why changing this seeders?

Reply:
@SimonImanigirimpuhwe
Since you created **SUPER_ADMIN** Role in (src/database/adminRole.create.js), we do not need to seed it again.
Thanks for your concern!